### PR TITLE
13721 Import latest shared components

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#13721

*Description of changes:*

- imported latest (Vue 2.6) shared components _(I did this to test the components before their update)_
- imported latest (Vue 2.7) shared components
- fixed Court Order POA layout
- app version = 5.7.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-    "@bcrs-shared-components/confirm-dialog": "1.1.0",
-    "@bcrs-shared-components/corp-type-module": "1.0.6",
-    "@bcrs-shared-components/court-order-poa": "1.0.14",
-    "@bcrs-shared-components/enums": "1.0.20",
-    "@bcrs-shared-components/interfaces": "1.0.40",
-    "@bcrs-shared-components/staff-comments": "1.2.6",
-    "@bcrs-shared-components/staff-payment": "2.0.10",
+    "@bcrs-shared-components/breadcrumb": "2.1.0",
+    "@bcrs-shared-components/confirm-dialog": "1.2.0",
+    "@bcrs-shared-components/corp-type-module": "1.0.7",
```
Comment (severinbeauvais) [inline package.json:21]: There was no Vue 2.7 update to `corp-type-module`.

File: package.json [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
+    "@bcrs-shared-components/court-order-poa": "2.1.0",
+    "@bcrs-shared-components/enums": "1.0.22",
```
Comment (severinbeauvais) [inline package.json:23]: There was no Vue 2.7 update to `enums`.

File: package.json [hunk 3]
```diff
@@ -16,14 +16,14 @@
...
+    "@bcrs-shared-components/interfaces": "1.0.42",
```
Comment (severinbeauvais) [inline package.json:24]: There was no Vue 2.7 update to `interfaces`.

File: src/components/common/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { BreadCrumb } from '@bcrs-shared-components/bread-crumb'
+import { Breadcrumb } from '@bcrs-shared-components/breadcrumb'
```
Comment (severinbeauvais) [inline src/components/common/index.ts:1]: Renamed in a recent version.

**Review by lambert-alex — APPROVED** [review]
LGTM!

**Review by lewischenstudio — APPROVED** [review]

**Pull request merged**